barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplonk.test.cpp

```diff
@@ -127,7 +128,7 @@ TYPED_TEST(ShplonkTest, ExportBatchClaimAndVerify)
         // KZG verifier
         auto final_proof_points =
             KZG<curve::BN254>::reduce_verify_batch_opening_claim(batched_verifier_claim, verifier_transcript);
-        ASSERT(this->vk().pairing_check(final_proof_points[0], final_proof_points[1]));
+        ASSERT_RELEASE(this->vk().pairing_check(final_proof_points[0], final_proof_points[1]));
```

Comment:
This is potentially expensive but since it's in a test, it's probably okay to run in release?

Reply:
Yeah all things with tests can be on